examples/timestamping/backend/src/lib.rs

```diff
@@ -37,11 +37,9 @@ pub mod transactions;
 
 use exonum::{
     merkledb::BinaryValue,
-    runtime::{
-        rust::{api::ServiceApiBuilder, CallContext, Service},
-        DispatcherError, ExecutionError,
-    },
+    runtime::{DispatcherError, ExecutionError},
```

Comment:
The Rust runtime should probably re-export frequently used types from the `runtime` module. Alternatively, there should be a facade crate reexporting the Rust runtime along with other components. (The main crate currently reexports `crypto` and `merkledb`, but it can't reexport the Rust runtime for obvious reasons.)